run.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+exec /app/dir/apipost_x64_8.1.4/apipost --no-sandbox
```

Comment:
if this is electron use `zypak` from the electron baseapp and drop `--no-sandbox`.